src/main/java/Tasks/Deadlines.java

```diff
@@ -0,0 +1,29 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+public class Deadlines extends Task {
+    private LocalDateTime deadlineInfo;
+    private static final DateTimeFormatter PATTERN_DISPLAY = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+    private static final DateTimeFormatter PATTERN_READ = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
```

Comment:
I like the use of SCREAMING_SNAKE_CASE for constant over here, which makes the code make a lot more readable